### PR TITLE
adding seqFISH multimodal package image

### PR DESCRIPTION
**Pull request by drighelli**

### CODE REVIEW

**Review by cgreene — APPROVED** [review]

**Comment by cgreene** [comment]
Did you want to add the figure to the manuscript as well? https://github.com/BIRSBiointegration/whitePaper/blob/master/USAGE.md#figures

**Comment by drighelli** [comment]
yes, I was waiting for the approval! 
Thanks!
:)

**Pull request merged**
